.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Asset
+
+on:
+    release:
+        types: [ created ]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Compile
+              run: php -d phar.readonly=0 compile-phar.php
+
+            - name: Upload
+              uses: actions/upload-release-asset@v1
+              env:
+                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                upload_url: ${{ github.event.release.upload_url }}
+                asset_path: ./exercise-cleaner.phar
+                asset_name: exercise-cleaner
```

Comment:
To hide the file extension may not be a good idea. It might be a choice to leave to the user. A little like in https://getcomposer.org/doc/00-intro.md#globally `mv composer.phar /usr/local/bin/composer`